google/data_source_stackdriver_uptime_check_ips.go

```diff
@@ -0,0 +1,55 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"sort"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/monitoring/v3"
+)
+
+func dataSourceStackdriverUptimeCheckIps() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceStackdriverUptimeCheckIpsRead,
+
+		Schema: map[string]*schema.Schema{
+			"uptime_check_ips": {
```

Comment:
Do you think other people that use this would want the region/location information as part of the data source too?